src/back/routes/update.ts

```diff
@@ -139,15 +145,11 @@ async function updateProposalUpdate(req: WithAuth<Request<{ update_id: string }>
       additional_notes,
       financial_records: parsedRecords,
     },
-    id,
-    proposal,
-    user!,
-    now,
-    isOnTime
+    user!
   )
 }
 
-async function deleteProposalUpdate(req: WithAuth<Request<{ update_id: string }>>) {
+async function deleteProjectUpdate(req: WithAuth<Request<{ update_id: string }>>) {
```

Comment:
If we are using projects I'd rather use the project_id here too. (you could also use the same validation mentioned before, see deletePersonnel i.e.)

Reply:
I think we don't need the project_id. As we can get the update and check if the user can edit it with update.project_id